stablehlo/dialect/StablehloOps.cpp

```diff
@@ -76,6 +76,9 @@ limitations under the License.
 #include "stablehlo/dialect/StablehloOps.h.inc"
 #include "stablehlo/dialect/TypeInference.h"
 
+#define GET_TYPEDEF_CLASSES
+#include "stablehlo/dialect/StablehloTypeDefs.cpp.inc"
```

Comment:
Can we put this right next to Enum / Attr includes?

Reply:
Ah is clang format moving this up? Not sure why `#include "stablehlo/dialect/StablehloOps.h.inc"` got moved up into the main include line, if we can move this down as well after all other headers

Reply:
Done.

No I put it there myself, sorry about the bad placement. I moved the Ops include down as well, PTAL.